tests/python/tank_test/tank_test_base.py

```diff
@@ -707,11 +800,11 @@ def create_file(self, file_path, data=""):
         open_file.write(data)
         open_file.close()
 
-    def check_error_message(self, Error, message, func, *args, **kws):
+    def check_error_message(self, error_type, message, func, *args, **kws):
         """
         Check that the correct exception is raised with the correct message.
 
-        :param Error: The exception that is expected.
+        :param error_type: The exception that is expected.
```

Comment:
Is this actually a class or a type? If the former the param should be `exception_class`?

Reply:
Why do we need this method in the first place? Why not using `self.assertRaisesRegexp`?

Reply:
Well... isn't a class a type? Is there some subtle difference I am not aware of?

I think this was introduce back in the day where we might have been using the unittest module, which in Python 2.5 and lower didn't have `assertRaisesRegexp`.